app/Policies/MessageThreadPolicy.php

```diff
@@ -6,33 +6,86 @@
 
 use App\Models\MessageThread;
 use App\Models\MessageThreadParticipant;
+use App\Models\Role;
 use App\Models\User;
 use Illuminate\Auth\Access\HandlesAuthorization;
 
 class MessageThreadPolicy
 {
     use HandlesAuthorization;
 
+    /**
+     * Role => display_name
+     */
+    protected const ROLE_INBOX_MAP = [
+        Role::ADMINISTRATOR => 'RAdmin',
+        Role::DEV_COMPLIANCE => 'DevCompliance',
+        Role::QUALITY_ASSURANCE => 'QATeam',
+        Role::ARTIST => 'RAArtTeam',
+        Role::CHEAT_INVESTIGATOR => 'RACheats',
+        Role::WRITER => 'WritingTeam',
+        Role::EVENT_MANAGER => 'RAEvents',
+    ];
```

Comment:
What about DevQuest, QualityQuest, TheUnwanted, and/or RANews?
https://retroachievements.org/manage/roles/23
https://github.com/RetroAchievements/RAWeb/blob/32443e37dbbb718fa0bd398a40d33a27230de48b/config/services.php#L38-L85

Reply:
These are tricky because (to my knowledge) we don't have any roles or permissions associated with them.

Short of creating new roles for these inboxes, I'm not sure how best to accommodate for this.